rust/gen/rust-bindings.h

```diff
@@ -0,0 +1,919 @@
+/* Copyright (C) 2017 Open Information Security Foundation
```

Comment:
should this be checked in at all? Seems more like something for the dist to contain a generated version of this from when the dist is built.